test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -520,6 +521,53 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
             }
         }
 
+        [Theory]
```

Comment:
> ~~Will write a unit test.~~

Since there's no `PathBase` left anywhere else but `Frame`, it's hard to write a unit test for this. A functional test is still the only place where I can specify a path base in `UseUrls` and verify Kestrel does nothing with it.

So either we leave no tests for this or we keep the functional test.